tests/bdd/features/lti_certification/section_1_invalid_launch_requests.feature

```diff
@@ -3,39 +3,20 @@ Feature: Section 1 - Invalid Launch Requests
   This section comprises tests using invalid launch requests.
 
   Background:
-    Given fixtures are located in '/lti_certification_1_1/section_1'
-      And the OAuth 1 consumer key is 'Hypothesis4dd96539c449ca5c3d57cc3d778d6bf3'
-      And the OAuth 1 shared secret is 'TEST_SECRET'
-      And I create an LMS DB 'ApplicationInstance'
-        | Field            | Value                                      |
-        | consumer_key     | Hypothesis4dd96539c449ca5c3d57cc3d778d6bf3 |
-        | shared_secret    | TEST_SECRET                                |
-        | lms_url          | test_lms_url                               |
-        | requesters_email | test_requesters_email                      |
-      And I load the fixture 'most_common.ini' as 'params'
+    Given standard setup for LTI section 1
 
   @v1.0 @v1.1 @v1.2 @required
   Scenario: Test 1.1 - No resource_link_id provided
     # Expected result: Return user to the Tool Consumer with an error message
 
     Given I update the fixture 'params' with
 	  | Key                     | Value               |
-	  | custom_link_setting_url | $LtiLink.custom.url |
 	  | resource_link_id        | *MISSING*           |
+      | custom_link_setting_url | $LtiLink.custom.url |
```

Comment:
This is me not knowing WTH `custom_link_setting_url` does, but why does it have to be set in addition to unsetting `resource_link_id` here?

Reply:
Once again, I have no idea, this is just what they did in the fixtures in the LTI Certification test. Honestly I think it's just noise, but I don't want to make that call.